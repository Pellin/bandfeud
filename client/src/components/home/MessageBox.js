import React from 'react';

const MessageBox = ({ message }) => {
  return <div className="message-box">{message}</div>;
};

export default MessageBox;
