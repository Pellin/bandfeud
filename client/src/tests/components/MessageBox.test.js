import React from 'react';
import { shallow } from 'enzyme';

import MessageBox from '../../components/home/MessageBox';

it('should render messageBox with set message', () => {
  const message = 'Get ready...';
  const wrapper = shallow(<MessageBox message={message} />);
  expect(wrapper.find('.message-box').text()).toBe(message);
  expect(wrapper).toMatchSnapshot();
});