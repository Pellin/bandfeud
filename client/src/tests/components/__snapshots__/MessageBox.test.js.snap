// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render messageBox with set message 1`] = `
<div
  className="message-box"
>
  Get ready...
</div>
`;
